### PR TITLE
Add index.html file and style.css file

### PR DESCRIPTION
**Pull request by lazyCode-404**
Things implemented in this milestone:
- Created a new empty repo
- Copied code from an old project "Grace-and-Bwalya-s-HTML-project"
- Created a new branch for the copied code
- Opened a pull request.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,293 @@
+<!DOCTYPE html>
+<html lang="en">
+
```
Comment (lazyCode-404) [inline index.html:3]: Please avoid leaving unnecessary space so that your code looks clean and neat.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+              globally or in the calling function.</p>
+            
+            
+            <h2 id="reference">Reference</h2>  
+            <p>All the documentation in this page is taken from MDN</p>
+            
+        </section>
+
+
+
```
Comment (BwalyaNonde) [inline index.html:288]: Please avoid leaving unnecessary space so that your code looks clean and neat.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+            followed by:</p>
+
+            <ul>
+                <li>The name of the function.</li>
+                <li>A list of arguments to the function, enclosed in parentheses and separated by commas.</li>
+                <li>The JavaScript statements that define the function, enclosed in curly brackets, { }.</li>
+                
+            </ul>
+            
+
```
Comment (BwalyaNonde) [inline index.html:262]: Please avoid leaving unnecessary space so that your code looks clean and neat.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,293 @@
...
+            while (condition) statement
+        </div>
+           
+          <p>If the condition becomes false, statement within the loop stops executing and control passes to the
+            statement following the loop.<br>
+            The condition test occurs before statement in the loop is executed. 
+            If the condition returns true, statement is executed and the condition is tested again. 
+            If the condition returns false, execution stops and control is passed to the statement following while.</p>
+
+
```
Comment (lazyCode-404) [inline index.html:229]: Please avoid leaving unnecessary space so that your code looks clean and neat.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,293 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```
Comment (lazyCode-404) [inline index.html:11]: kindly check the script tag and put it in the bottom part of the body section for the best practice of html semantic.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,293 @@
...
+                <p>The scope rules for constants are the same as those for let block scope variables. If the const keyword is omitted, the identifier is assumed to represent a variable.</p>
+                <p>You cannot declare a constant with the same name as a function or variable in the same scope. For example:</p>
+            </p>
+            <div class="functions">
+                <p>// THIS WILL CAUSE AN ERROR function f() {}; const f = 5; // THIS WILL CAUSE AN ERROR ALSO function f() { const g = 5; var g; //statements }
+                </p>
+            </div>
+            <p>However, object attributes are not protected, so the following statement is executed without problems.</p>
+            <div class="functions">
+                const MY_OBJECT = {"key": "value"}; MY_OBJECT.key = "otherValue";
```
Comment (lazyCode-404) [inline index.html:151]: Kindly try to implement this line using a span rather than a div for the best practice.

File: index.html (hunk 3 above)
Comment (lazyCode-404) [inline index.html:262]: Please avoid leaving unnecessary space so that your code looks clean and neat.

File: index.html (hunk 2 above)
Comment (lazyCode-404) [inline index.html:288]: Please avoid leaving unnecessary space so that your code looks clean and neat.

**Comment by lazyCode-404** [comment]
Kindly work on the css file to apply styling to your project
Please update the README. file so that it makes your project look professional

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @BwalyaNonde,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
But Kindly consider the [Optional] suggestion.



## (Highlights) Good Points: :+1:
- You used inline comments
- You asked for changes in a polite way.
- You mentioned the reason why the reviewee had to do your required changes.

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- Wow your inline comments are great but next time Kindly make all your inline comments inside one review not like you did each inline comment in a separate review.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by lazyCode-404** [comment]
Thank you so much for approving my project.

**Pull request merged**